BUILDING.md

```diff
@@ -265,16 +265,15 @@ More Developer Tools...`. This step will install `clang`, `clang++`, and
 
 #### Building Node.js
 
-If the path to your build directory contains a space, the build will likely
-fail.
-
 To build Node.js:
 
 ```console
 $ ./configure
 $ make -j4
 ```
 
+Remember to contain the path to your file with single or double quotes. spaces without quoations are interpreted by bash as seperate commands, thus they will fail (or do something rediculous).
```

Comment:
This isn't correct. As you can see on lines 271 and 272, the path isn't supplied. There's nothing to put quotes around. This is not a helpful change.